my-first-react-app/src/App.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Hello from './Hello.js';
+
+class App extends React.Component {
+  render() {
+    const users = [
+      {_id: 1, name: 'Leia'},
+      {_id: 2, name: 'Luke'},
+      {_id: 3, name: 'Han'}
+    ];
+
+    return (
+      <div>
+        {users.map(user => <Hello key={user.id} name={user.name} />)}
```

Comment:
Hey Danny, this is looking good!

Though, `user.id` should be `user._id` 

`user.id` would be undefined always because our users are structured like this `{_id: Number, name: String}`.